app/representers/api/series_representer.rb

```diff
@@ -9,6 +9,7 @@ class Api::SeriesRepresenter < Api::BaseRepresenter
   property :created_at, writeable: false
   property :updated_at, writeable: false
   property :app_version, writeable: false
+  property :import_url
```

Comment:
I wonder if there's any case for only exposing the original-feed-url in the auth-series-representer

Reply:
Hmm, not sure I understand. `original-feed-url` as in "this feed url is where this series was derived from"?

The representer here really bears the burden of the underlying resources.

Reply:
Just wondering if there's a security issue with telling the world where our feeds originally came from.  Then again, feeds are public so why would we care?  Wondering withdrawn!

And yeah, if this moved to the auth representer, you couldn't set it via the public controller.